static/app/views/detectors/components/details/common/ongoingIssues.tsx

```diff
@@ -1,19 +1,67 @@
+import {LinkButton} from 'sentry/components/core/button/linkButton';
+import {Flex} from 'sentry/components/core/layout';
+import EmptyStateWarning from 'sentry/components/emptyStateWarning';
 import ErrorBoundary from 'sentry/components/errorBoundary';
-import {SimpleTable} from 'sentry/components/tables/simpleTable';
+import GroupList from 'sentry/components/issues/groupList';
+import Panel from 'sentry/components/panels/panel';
+import PanelBody from 'sentry/components/panels/panelBody';
 import Section from 'sentry/components/workflowEngine/ui/section';
 import {t} from 'sentry/locale';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import useOrganization from 'sentry/utils/useOrganization';
+
+interface Props {
+  detectorId: string;
+  query?: Record<string, any>;
+}
+
+function EmptyMessage() {
+  return (
+    <Panel>
+      <PanelBody>
+        <EmptyStateWarning small withIcon={false}>
+          {t('No issues found for this monitor')}
+        </EmptyStateWarning>
+      </PanelBody>
+    </Panel>
+  );
+}
+
+export function DetectorDetailsOngoingIssues({detectorId, query}: Props) {
+  const organization = useOrganization();
+
+  const queryParams = {
+    ...query,
+    query: new MutableSearch(['is:unresolved', `detector:${detectorId}`]).formatString(),
```

Comment:
do we expect this to be more than just `is:unresolved detector:1` i think just a string probably works here